react-graphql-client-library.md

```diff
@@ -23,31 +23,31 @@ summary = "A tutorial on how to build your own GraphQL client for React applicat
 
-* Second, there must be a way to **provide the GraphQL client instance to the React view layer**. It is the perfect use case for [React's Context API](https://www.robinwieruch.de/react-context-api/) to provide the GraphQL client instance somewhere at the top level of your React component tree and to consume it in every React component which is interested in it.
+2. There must be a way to **provide the GraphQL client instance to the React view layer**. It is the perfect use for [React's Context API](https://www.robinwieruch.de/react-context-api/) to provide the GraphQL client instance at the top level of the React component tree, and to consume it in every React component interested in it.
 
-* Third, there must be a way to **execute GraphQL operations, such as a query or a mutation, in a declarative way in React**. Therefore, you will implement a Query component and a Mutation component which will expose an API to execute the GraphQL operations and to access the result of it. Because you are implementing these components, you will never touch the GraphQL client which is provided with React's Context API explicitly in your React components, but only in the Query and Mutation components which are implemented by you.
+3. There must be a way to **execute GraphQL operations, like a query or a mutation, in a declarative way in React**.  You will implement a Query component and a Mutation component that exposes an API to execute the GraphQL operations and to access its result. Because you are implementing these components, you won't touch the GraphQL client provided with React's Context API explicitly in your React components, but only in the Query and Mutation components.
 
-Whereas the first part of the previously described steps is React agnostic, the second and third parts glue the GraphQL client (data layer) to React (view layer). You can see it analog to the *redux* and *react-redux* or *apollo-client* and *react-apollo* libraries. Whereas the former part is view layer agnostic, the latter part is used to connect it to the view layer.
+The first part is React agnostic, but the second and third glue the GraphQL client (data layer) to React (view layer). Compare to the *redux* and *react-redux* or *apollo-client* and *react-apollo* libraries. The former is view layer agnostic, the latter is used to connect it to the view layer.
 
-Along the way of implementing your own GraphQL client for React in the following sections, you will implement a simple GitHub client application with React which consumes GitHub's GraphQL API by using your own GraphQL client. So let's get started.
+While you mplement a GraphQL client for React in the following sections, you will also implement a GitHub client application with React that consumes GitHub's GraphQL API, using GraphQL client. 
```

Comment:
"mplement" to "implement"

Reply:
"using GraphQL client" to "using your GraphQL client"

Reply:
The second part:
I remove a lot of instances in these chapters where the reader is referenced directly without good reason.  This is a good example.  You already addressed the reader twice in this sentence: "implementing your own GraphQL..." and "you will implement".  They are already aware that you're referencing them, so the "your" at the end is implicit.